inc/3rd-party/plugins/i18n/wpml.php

```diff
@@ -1,11 +0,0 @@
-<?php
```

Comment:
Need to also remove the `require` for this file in `inc/3rd-party/3rd-party.php`